metafix/src/main/java/org/metafacture/metafix/FixMethod.java

```diff
@@ -84,6 +85,24 @@ public void apply(final Metafix metafix, final Record record, final List<String>
             metafix.putMap(params.get(0), options);
         }
     },
+    put_rdfmap {
+        @Override
+        public void apply(final Metafix metafix, final Record record, final List<String> params, final Map<String, String> options) {
+            final String fileName = params.get(0);
+            final RdfMap rdf = new RdfMap();
+            rdf.setFile(metafix.resolvePath(fileName));
+            if (options.containsKey("targetLanguage")) {
+                rdf.setTargetLanguage(options.get("targetLanguage"));
```

Comment:
I know this pull request is still not ready for review, but one minor thing: Options to Fix functions are written in _snake_case_, not _camelCase_.

Reply:
changed that